tests/test_codebase/test_mmpose/data/yolox-pose_s_8xb32-300e_coco.py

```diff
@@ -0,0 +1,259 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+_base_ = ['_base_/default_runtime.py']
+
+# model settings
+model = dict(
+    type='YOLODetector',
+    use_syncbn=False,
+    init_cfg=dict(
+        type='Pretrained',
+        checkpoint='https://download.openmmlab.com/mmyolo/v0/yolox/'
```

Comment:
could we set checkpoint to `None`. No need to download the ckpt each time run the ut.